protoc-gen-swagger/genswagger/template.go

```diff
@@ -13,6 +13,49 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor"
 )
 
+// messageToQueryParameters converts a message to a list of swagger query parameters.
+func messageToQueryParameters(message *descriptor.Message, reg *descriptor.Registry, pathParams []descriptor.Parameter) ([]swaggerParameterObject, error) {
+	var parameters []swaggerParameterObject
+	var addParameterWithPrefix func(string, *descriptor.Field) error
```

Comment:
```go
addParameterWithPrefix := func(prefix string, field *descriptor.Field) error {
```

BTW, why do you need to keep this function nested?
IIUC, you could simply define a function like
```go
func queryParams(prefix string, field *descriptor.Field) ([]swaggerParameterObject, error)
```